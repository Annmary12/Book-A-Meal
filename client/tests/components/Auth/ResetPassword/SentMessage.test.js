import React from 'react';
import SentMessage from '../../../../src/components/Auth/ResetPassword/SentMessage';

describe('SentMessage component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<SentMessage />);

    expect(wrapper).toMatchSnapshot();
  })
})