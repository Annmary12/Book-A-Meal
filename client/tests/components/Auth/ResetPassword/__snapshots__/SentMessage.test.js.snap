// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SentMessage component should match snapshot 1`] = `
<div
  className="mailSent"
>
  <h4>
    Reset password link would be sent to your email shortly.
  </h4>
  <p>
    Go back to 
    <Link
      replace={false}
      to="login"
    >
      Login
    </Link>
  </p>
</div>
`;
